analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -1003,11 +1006,16 @@ def parse_options(compilation_db_entry,
     ImplicitCompilerInfo.compiler_versions[details['compiler']] \
         = compiler_version_info
 
-    using_clang_to_compile_and_analyze = False
-    if ImplicitCompilerInfo.compiler_versions[details['compiler']]:
+    using_same_clang_to_compile_and_analyze = False
+    compiler_clang = \
+        ImplicitCompilerInfo.compiler_versions.get(details['compiler'],
+                                                   False)
+    if compiler_clang:
         # Based on the version information the compiler is clang.
-        using_clang_to_compile_and_analyze = True
         flag_processors = clang_flag_collectors
+        if compiler_clang.installed_dir == analyzer_clang.installed_dir:
+            LOG.debug("Same clang is used for compilation and analysis.")
```

Comment:
I logged my dummy project with `/usr/bin/clang` and analyzed with a different clang `...CodeChecker/cc_bin/clang` but I still see this debug message in my output.

Reply:
are the installDir values the same for both of the clang, is it possible that the built package still used the clang from the PATH, I did not experience this, but I will double check.